### DISCUSSION TITLE
Is it possible to use different coin to check proposal validation?

### DISCUSSION THREAD

**Discussion by obbaeiei** [snapshot-labs/snapshot#1391] [Q&A] [answered]
For example, Users must be hold 1st ERC721 strategy to vote. But 2nd ERC721 to create proposal.

**Comment by bonustrack** [comment, marked as answer]
@obbaeiei Yes it's possible using basic validation parameters, you will need to input the strategies like: 
```json
{
  "strategies": [
    {
      "name": "erc721",
      "params": {
        "address": "0x25ed58c027921e14d86380ea2646e3a1b5c55a8b"
      }
    }
  ]
}
```

Another option is to use a custom validation strategy, it's not documented but you can check the code here for others strategies: https://github.com/snapshot-labs/snapshot.js/tree/master/src/validations

**Reply by RogerPodacter** [reply]
Is this answer still correct? When I try these exact parameters my proposal creation is still validated against the first strategy in my space.

**Reply by samwhosung** [reply]
@RogerPodacter after extensive testing I can confirm that it works using the example above yes. But the strategy you add to validation will only work with the main network that you have set in the space settings.
